.github/workflows/prune-images.yml

```diff
@@ -17,7 +17,7 @@ on:
         description: "Comma seperated list of application Image Streams to prune"
         required: true
         default: "web-sims, api-sims, queue-consumers-sims, workers-sims"
-      jobs:
+      ocjobs:
```

Comment:
Can it follow the same camel case pattern from `minTags` which would be `ocJobs`?